src/UXCapture.js

```diff
@@ -45,6 +47,8 @@ const UXCapture = {
 	create: config => {
 		_onMark = config.onMark || NOOP;
 		_onMeasure = config.onMeasure || NOOP;
+		_markSelector = config.markSelector;
```

Comment:
What's the difference between `markSelector` and `elementSelector`? They look very similar in practice, and I'm wondering if we need both.

Reply:
Agreed, see above.